agent/config/config.go

```diff
@@ -188,7 +188,7 @@ func get(args []string, l *logrus.Entry) (cfg *Config, configFileF string, err e
 
 		// set default values
 		if cfg.ListenAddress == "" {
-			cfg.ListenAddress = "127.0.0.1"
+			cfg.ListenAddress = "0.0.0.0"
```

Comment:
@rishat-ishbulatov can you please elaborate why we're changing the default listen address to all interfaces and exposing it to public?

Reply:
```    Open PMM-Agent_URL/metrics in browser
    Check that pmm-agent exposes all these metrics
    Check that agent_id label is exist.

Reply:
This exposes the whole `pmm-agent` API though. We shall figure out a different way to do this.

Reply:
ok then i'll revert changes